src/firefly/js/visualize/ui/SearchSummary.jsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {flux} from '../../Firefly.js';
+import {parseWorldPt} from '../Point.js';
+import numeral from 'numeral';
+
+export class SearchSummary extends PureComponent {
```

Comment:
Let's move this to js/ui instead of visualize.  visualize is mostly image related.

Reply:
ok